packages/api/src/indexPriceInterface.ts

```diff
@@ -48,7 +48,11 @@ export default abstract class IndexPriceInterface extends Observer {
 
 		sdkInterface.registerObserver(this);
 		this.sdkInterface = sdkInterface;
-		// trigger exchange info so we get an "update" message
+		// trigger exchange info so we get an "update" message.
+		//
+		// 2024-07-02: this exchangeInfo call does not actually issue the
+		// "update" call, since initialization is not yet done in
+		// EventListener when priceInterfaceInitialize is called.
 		const info = await this.sdkInterface.exchangeInfo();
```

Comment:
so should we remove it then?

Reply:
no, the info is needed for `_initIdxNamesToPerpetualIds`